### ISSUE TITLE
Licence?

### ISSUE THREAD

**Issue by AvgZing** [BoxOfDevs/SwitchFromLeet#2]
No licence means that they cannot **use** the software. :/ BRO.

**Comment by HimbeersaftLP** [comment]
It'a a website, nobody else other than us should host it :P

**Comment by AvgZing** [comment]
Why not? He (you know who) did it all the time!

**Comment by Vaults1** [comment]
You know who?
![img_2920](https://cloud.githubusercontent.com/assets/6726237/23901851/0dd13d22-08d0-11e7-9dee-af4cf3ccc5c3.PNG)

**Comment by AvgZing** [comment]
No, the other cat guy. Not you.

**Comment by lavender-sys** [comment]
@TheRoyalBlock You mean Stampy?

**Comment by AvgZing** [comment]
HOW DO YOU GUYS NOT GET THIS- I am talking about the old HIMBEER.

**Comment by HimbeersaftLP** [comment]
So I'm old huh?

**Comment by AvgZing** [comment]
yoy  I see u edited that.

**Comment by AvgZing** [comment]
Why did you keep editing my posts? Original post: Edited. Post 3: Edited. I'm sure there is a good reason

**Comment by HimbeersaftLP** [comment]
> So I'm old huh?

I'd never say something like that xD

**Comment by Vaults1** [comment]
The old himbeer? Lol. YEAH

**Comment by Vaults1** [comment]
Oops, sorry guys. I forgot to logout, it was my friend.

**Comment by HimbeersaftLP** [comment]
Bad excuse xD

**Comment by Vaults1** [comment]
@HimbeersaftLP Fine... Oops sorry guys it was Donald Trump he accidentally fired nuclear weapon at my house so he sent the FBI to come but they ended up in Greece instead then they reported me because my house was too far for them so they came and toook my iPad and then gave it to him and he didn't know what an iPad is so he logged on and messed up my github. **Better?**

**Comment by HimbeersaftLP** [comment]
Yep, that's something I'd believe xD

**Comment by InspectorGadget** [comment]
@Ultie0 You use GitHub to commit on an iPad?

**Comment by AvgZing** [comment]
@RTGDaCoder you got the words mixed up lol but sure, why not? I use ant device that is available

**Comment by InspectorGadget** [comment]
Don't! I learned my lesson when i used to use an iPad for commits xD

**Comment by hcortezr** [comment]
Boi yall are dumb.. Just cuz leet used your plug.. You should be happy 😂

**Comment by Ad5001** [comment]
@eDroiid Duh. It's easy to see that you're NOT an free dev... Lets see when they'll steal your work...

**Comment by Ultie0** [comment]
@eDroiid you clearly don't know the definition of dumb.

**Comment by Vaults1** [comment]
@RTGDACODER I use it for rousing github, but not commits.

**Comment by ghost** [comment]
@eDroiid LEET used their plugin and violated the license, how should they be happy? Honestly, I like LEET and its community, but it's not right to steal a plugin and violate the license like that.
Sure, BoxOfDevs is maybe going a little too far by sending someone to spam the forums, when they should do things professionally, and that's not improving the situation.
If you guys went and talked to LEET professionally and respectfully in the first place, they would have probably listened.

**Comment by AvgZing** [comment]
@Echodoris we were joking about the spam XD

**Comment by Ad5001** [comment]
@Echodoris It was a joke about spam. And when we talk about professionalism, we tried being kind, but that didn't solved the problem at all. They were ignoring the evident.

**Comment by HimbeersaftLP** [comment]
> Sure, BoxOfDevs is maybe going a little too far by sending someone to spam the forums, when they should do things professionally, and that's not improving the situation

We never did that...

**Comment by lavender-sys** [comment]
@Echodoris We never sent anyone to spam the forums. We did announce on the forums to get their attention, but that didn't work, so now comes the DMCA.

**Comment by ghost** [comment]
@HimbeersaftLP You went on the forums and said a bunch of unecessary (true) crap about LEET
So now with how you guys acted they're not taking you seriously and they think of you as "baboons"

**Comment by InspectorGadget** [comment]
You guys had a GL License when you've pushed your commits to your Repo? If yes, GL license means they can use it but no duplication or redistribution

**Comment by lavender-sys** [comment]
Then they redistributed to their customers.

**Comment by InspectorGadget** [comment]
That could be the case too

**Comment by lavender-sys** [comment]
http://forum.leet.cc/topic/24789/announcement-news-alert-for-leet-forums/13

**Comment by ghost** [comment]
The original license stated (I believe at line 38) that if the software is distributed, the end user must be given a copy of the source code - which LEET isn't doing - and they must also be informed that they have the same rights over the software as the distributor does.
Basically the end user must be given the code and must be told they have the right to use the software for production, distribute it, sell it, and all the other rights included in the license.
And LEET is violating both of these.

**Comment by AvgZing** [comment]
I'm really confused: whose side are you on, @ThouvZePig?

**Comment by HimbeersaftLP** [comment]
> @HimbeersaftLP You went on the forums and said a bunch of unecessary (true) crap about LEET
So now with how you guys acted they're not taking you seriously and they think of you as "baboons"

I was trying to make at least a few members move from leet. Oh, funfact: They banned even tough I wasn't violating the ToS xD

**Comment by dragonwocky** [comment]
> I'm really confused: whose side are you on, @ThouvZePig?

Ours, but he doesn't approve of how we acted by posting all that stuff on the Leet forums, even tho he accepts it was true.

**Comment by Vaults1** [comment]
I contacted leet support. Let's see what they do.

**Comment by ghost** [comment]
@TheRoyalBlock I agree with you guys about the situation itself, but I agree with LEET when they say that you acted immaturely about it

**Comment by Thunder33345** [comment]
> The original license stated (I believe at line 38) that if the software is distributed, the end user must be given a copy of the source code - which LEET isn't doing - and they must also be informed that they have the same rights over the software as the distributor does.
Basically the end user must be given the code and must be told they have the right to use the software for production, distribute it, sell it, and all the other rights included in the license.
And LEET is violating both of these.

I think you are right, based on the old license which they choose to obligate to, they are cornered to share their source code basically

**Comment by dragonwocky** [comment]
@ThouvZePig I can't argue with that, I agree. I did just try and talk to one the devs about it in a DM/PM/Whatever-You-Want-To-Call-It-Message, but he refused to negotiate at the time. More recently I tried messaging him again and he ignored me and tried to redirect me to the Leet support email.

@Thunder33345 Exactly.

**Comment by Thunder33345** [comment]
when will we get this project going tho?

**Comment by HimbeersaftLP** [comment]
> when will we get this project going tho?

When I'm finished with my other projects :D

(BOD forums Discord WebHook and CommandShop)

**Comment by InspectorGadget** [comment]
xD. I'm working on Forums... It has avatar upload and also with SQL Data handling. I can help with this also if this has any login forms or any other feature regarding PHP. It can be Super Globals too :)

**Comment by Thunder33345** [comment]
>Super Globals

?? also you are building your own forum?!?

>Discord WebHook

wym by that?

**Comment by HimbeersaftLP** [comment]
> wym by that?

A WebHook that writes a message in #status when a new forum message is posted

**Comment by InspectorGadget** [comment]
@Thunder33345 Yes, why not?

**Comment by Thunder33345** [comment]
@RTGDaCoder seems too much when there's free alternatives out there that's secure and more completed

**Comment by InspectorGadget** [comment]
@Thunder33345 I've nothing to work on so i'm doing that for my new service
